api/v1/swagger.yaml

```diff
@@ -1790,6 +1790,586 @@ paths:
+              - createdAt
+              - description
+              - helpEmail
+              - helpUrl
+              - id
+              - name
+              - updatedAt
+              - - isActive
+              - - keywords
+              - - aspects
+              - - subjects
+              - - subjectQuery
+              - - createdAt
+              - - description
+              - - helpEmail
+              - - helpUrl
+              - - id
+              - - name
+              - - updatedAt
+        -
+          $ref: "#/parameters/limitParam"
+        -
+          $ref: "#/parameters/offsetParam"
+        -
+          name: description
+          in: query
+          type: string
+          maxLength: 4096
+          description: >
+            A description of the generator.
+        -
+          name: helpEmail
+          in: query
+          type: string
+          description: >
+            The email address where a user can go to get more help about the generator.
+          maxLength: 254
+        -
+          name: helpUrl
+          in: query
+          type: string
+          maxLength: 2082
+          description: >
+            The url where a user can go to get more help about the generator.
+        -
+          name: createdBy
+          in: query
+          type: string
+          description: >
+            Id of the User who created this Generator.
+        -
+          name: id
+          in: query
+          type: string
+          description: >
+            The collector id.
+        -
+          name: isActive
+          in: query
+          type: boolean
+          description: Whether or not the generator is running on a collector.
+        -
+          name: name
+          in: query
+          type: string
+          description: The generator's name.
+        -
+          name: subjectQuery
+          in: query
+          type: string
+          description: The query to append to GET subjects.
+        -
+          name: subjects
+          in: query
+          description: AbsolutePaths of the subjects.
+          type: array
+          items:
+            type: string
+        -
+          name: aspects
+          in: query
+          description: Names of the aspects.
+          type: array
+          items:
+            type: string
+        -
+          name: keywords
+          in: query
+          description: Describes this generator.
+          type: array
+          items:
+            type: string
+      responses:
+        200:
+          description: >-
+            Success, returns a list of generators
+          schema:
+            type: array
+            items:
+              $ref: "#/definitions/GeneratorResponse"
+        400:
+          $ref: "#/responses/400"
+        default:
+          $ref: "#/responses/genericError"
+    post:
+      security:
+        - jwt: []
+      summary: Create a new generator.
+      tags: [ generators ]
+      description: >-
```

Comment:
yep fixed. The changes should only have "description: >-" instead of "description: >"